python-basic-data-types/point.py

```diff
@@ -0,0 +1,15 @@
+class Point:
+    def __init__(self, x, y):
+        self.x = x
+        self.y = y
+
+    def __bool__(self):
+        if self.x == self.y == 0:
+            return False
+        return True
+
+
+origin = Point(0, 0)
+bool(origin)
+point = Point(2, 4)
+bool(point)
```

Comment:
I'd suggest to make this executable with some output, either like this:

```suggestion
origin = Point(0, 0)
print(bool(origin))
point = Point(2, 4)
print(bool(point))
```

Or, potentially wrap it into the name-main idiom, but maybe no need for that for this.